src/app/service/common-tools.service.ts

```diff
@@ -1,15 +1,25 @@
 /*
- * Copyright © Live-Poll 2020. All rights reserved
+ * Copyright © Live-Poll 2020-2021. All rights reserved
  */
 
 import {Injectable} from '@angular/core';
 import {Poll} from '../model/poll';
+import {HttpClient} from '@angular/common/http';
+import {NzNotificationService} from 'ng-zorro-antd/notification';
 
 @Injectable({
   providedIn: 'root'
 })
 export class CommonToolsService {
 
+  /**
```

Comment:
sad that GitHub doesn't provide code formatting for this typescript file, don't know why, maybe because it's called `....service.ts`. But it works for `....component.ts`

Reply:
Don't know either. Maybe you are right